static/src/javascripts/projects/common/modules/navigation/sticky-social-buttons.js

```diff
@@ -0,0 +1,55 @@
+define([
+    'fastdom',
+    'common/utils/$',
+    'common/utils/_',
+    'common/utils/mediator'
+], function (
+    fastdom,
+    $,
+    _,
+    mediator
+) {
+
+    var selectorTopEl = '.social--top',
+        selectorNavSwapOutEl = '.l-header-pre',
+
+        deadzone = 50,
+
+        topEl = _.memoize(function () { return $(selectorTopEl)[0]; }),
```

Comment:
would plain `$(mySelector)[0]` do the same without needing to run `_.memoize()`?